src/main/java/net/pms/remote/RemoteWeb.java

```diff
@@ -293,8 +296,9 @@ public void handle(HttpExchange t) throws IOException {
 				r.checkThumbnail();
 				in = r.getThumbnailInputStream();
 			}
+			in = ImagesUtil.convertImage(in, ImageFormat.JPEG);
 			Headers hdr = t.getResponseHeaders();
-			hdr.add("Content-Type", r.getThumbnailContentType());
+			hdr.add("Content-Type", HTTPResource.JPEG_TYPEMIME);
```

Comment:
Webp is too new for me, I don't know how well supported it is. I had PNG first, but I changed it because of memory consumption. When I looked at typical image thumbnails generated by ```IOUtils```, the PNG's were often close to 10x the size. Given the GC's mysterious ways, there's no way to know how long we'll actually have to live with these objects in memory, and they might take less memory in the browser as well (I don't know in what form the browsers store them).

The point is just that we have to force something as long as we don't know what it is. The old logic was broken, that's why I removed it. What ```getThumbnailContentType()``` returned was mostly hardcoded in the individual class, and had nothing to do with the actual format of in-memory thumbnail. Thus we often told the browser it got one image format and sent another. Luckily it mostly worked, the browsers probably "detect" it by the bytes instead of taking our word for it. This is the only easy way to give a correct mime type.

The whole thumbnail handling could have been improved further if we stored a few metadata about the thumbnail (format, width and height would go a long way), but given how many different sources there are for them that's no small feat and I didn't want this to grow even bigger.